### ISSUE TITLE
Errors trying to build latest Q2RTX

### ISSUE THREAD

**Issue by BoostAddicted** [NVIDIA/Q2RTX#302]
I attempted to follow the build instructions (https://github.com/NVIDIA/Q2RTX#build-instructions) using CMake (3.27.2) GUI and Visual Studio (2022) on my Windows 11 PC, and I was met with some errors.  I'm not familiar with these apps (or a dev), but I'd like to learn how to build the latest versions of Q2RTX periodically as updates are made (but prior to publishing to Steam).  Am I doing something incorrectly?

![image](https://github.com/NVIDIA/Q2RTX/assets/97083348/53ebc747-834c-4e5d-bc05-8c9845d9b370)

![image](https://github.com/NVIDIA/Q2RTX/assets/97083348/c6a571ae-6443-4302-95f3-a00e89bced5e)

![image](https://github.com/NVIDIA/Q2RTX/assets/97083348/a088a602-407e-4f21-a344-fb9f4cd89f04)

**Comment by apanteleev** [comment]
It looks like you need to either install the Vulkan SDK, or build glslangValidator from source - it's included here as a submodule.

Vulkan SDK can be found at https://www.lunarg.com/vulkan-sdk/
To build glslangValidator, check the "CONFIG_BUILD_GLSLANG" setting in the CMake configuration.

**Comment by BoostAddicted** [comment]
Looks like it was in fact the Vulkan SDK missing that caused my errors - thanks.  I was able to finish the CMake process and build in Visual Studio, but I can't seem to get the game to run. I tried to figure it out on my own, but I'm probably missing a step.  I'm not sure what step 3 is asking me to do (I know what a symbolic link is).  I'm guessing I don't have the necessary game files within the folders. I feel bad asking, but hoping someone can help me learn.

![image](https://github.com/NVIDIA/Q2RTX/assets/97083348/0147658b-e6ee-4884-b707-1a657f7043d4)

**Comment by CyberAP** [comment]
I'd suggest building with Visual Studio 2022 (don't forget to install CMake extensions in Visual Studio installer). I tried builing just using cmake but that didn't work for me either. It did work through the Visual Studio app with this guide: https://learn.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-170

**Comment by res2k** [comment]
Try to get the "standard output", for example, by running Q2RTX.exe from inside Visual Studio, which should capture the output. Any errors etc. being reported may give a good hint on what you're missing.

Alternatively, you could also attach a debugger to the process when that dialog pops up and press "Retry".
Investigating the state of the application may also tell you what's not working ... However, this is somewhat more involved; try to get the aforementioned output first, and if that isn't enough, dig deeper with the debugger.

Last but not least, you can just guess - if you look into the Steam release you can notice Q2RTX comes with some extra files (`blue_noise.pkz`, `q2rtx_media.pkz`) beyond the Quake 2 data files, so you could try to just copy those into your "build" folder.

**Comment by BoostAddicted** [comment]
Thanks @res2k. I was missing `blue_noise.pkz` and `q2rtx_media.pkz` in my build path: F:\Steam\steamapps\common\Quake II RTX\baseq2\Q2RTX-Github\Q2RTX\baseq2.  I had them in the wrong place.

After building with Visual Studio, I needed to copy `pak0.pak`, `blue_noise.pkz`, and `q2rtx_media.pkz` to the above directory for the game to launch.  I also noticed the `players` and `video` folders in Q2RTX/baseq2 missing. `Music` was also copied.  Unsure if anything else is missing - I have yet to see any errors in the console.

I found the aimfix (https://github.com/NVIDIA/Q2RTX/issues/274) within the Player menu.  Is it more appropriate to have that in the input or other menu?
